selene.toml

```diff
@@ -0,0 +1,19 @@
+std = "roblox+testez"
+
+[config]
+empty_if = { comments_count = true }
+unused_variable = { ignore_pattern = "result|ok|^_" }
+# this comes up when translating nested try/finally scenarios
```

Comment:
Looks like there's some settings and comments here from another project that likely don't apply; could you double check this file and clean those up? We may not need some of these particular settings to be non-default.